mkt/webapps/tasks.py

```diff
@@ -386,33 +366,6 @@ def dump_apps(ids, **kw):
         dump_app(id)
 
 
-@task
-def zip_apps(*args, **kw):
```

Comment:
zip_apps() was called by `process_addons --task=dump_apps` which we have not used in a while. `export_data` command does this directly.